README.md

```diff
@@ -1,12 +1,15 @@
-![banner](https://raw.githubusercontent.com/Hephaestus-Dev/FibLib/master/banner.png "banner")
+<img src="https://raw.githubusercontent.com/Hephaestus-Dev/FibLib/master/banner.png" width="300" height="150" alt="Banner">
+
 # FibLib ![latest version](https://img.shields.io/github/v/release/Hephaestus-Dev/FibLib)
 This is a library all about lying to your players! 
 
 ## Setup
-For setup instructions please see the [FibLib wiki](https://github.com/Hephaestus-Dev/FibLib/wiki).
+For setup instructions please see the [wiki.](https://github.com/SkyfallWasTaken/FibLib/wiki)
```

Comment:
Let's change the link to https://github.com/Haven-King/FibLib/wiki & transfer your wiki over to this repository.

Reply:
Done